apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-auto-complete-validation-example.component.html

```diff
@@ -21,13 +15,9 @@
     <div *ngIf="!itemTemplate" class="fd-popover__body fd-popover__body--no-arrow">
         <nav class="fd-menu">
             <ul fd-menu-list class="fd-menu__list">
-                <li
-                    fd-menu-list-item
-                    class="fd-menu__item"
-                    *ngFor="let option of options"
-                    (click)="onItemClick(option)"
-                >
-                    {{ option }}
+                <li fd-menu-list-item class="fd-menu__item" *ngFor="let option of options"
+                    (click)="onItemClick(option)">
+                    <span href="#" class="fd-menu__link">{{ option }}</span>
```

Comment:
we shouldn't use `href` attribute on `span`, it's not a valid html

Reply:
I has been removed.